common/lib/common-utils/pnpm-lock.yaml

```diff
@@ -830,7 +831,7 @@ packages:
     engines: {node: '>= 10.14.2'}
     dependencies:
       '@jest/types': 26.6.2
-      '@types/node': 15.14.9
+      '@types/node': 14.18.42
```

Comment:
Not sure if this PR should do it, but we should update `pnpm->overrides->@types/node` in the root `package.json` to 16.x, right? Server already uses 16.

Reply:
We want the types to match the lowest version of node we support (currently 14). Server bumped to 16 types before they should have.